chirp/drivers/ft450d.py

```diff
@@ -304,8 +306,8 @@ class FT450DRadio(yaesu_clone.YaesuCloneModeRadio):
         struct mem_struct current;
 
     """
-    _CALLSIGN_CHARSET = [chr(x) for x in range(ord("0"), ord("9") + 1) +
-                        range(ord("A"), ord("Z") + 1) + [ord(" ")]]
+    _CALLSIGN_CHARSET = [chr(x) for x in itertools.chain(range(ord("0"), ord("9") + 1),

```

Comment:
I don't understand the need for complication here (I know you didn't originate this code). I suggest using a literal here

```python3
_CALLSIGN_CHARSET = [
    '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 
    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 
    'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 
    ' ']
```

Reply:
We do this in a lot of places to save the typing, but also to make sure we don't miss any. I know it should be trivial, but I think there's a reasonable argument for having it be generated.

Either way, best not to change fundamentals like this in a patch that purports to just be compatibility-related.